.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
I don't think the higher boot times have a marginal effect on the already minute long travis run anyway ;)